packages/react-native-select-pro/src/components/select-control/select-control.hooks.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
+import { AccessibilityInfo } from 'react-native';
+
+import { isAndroid } from '../../helpers';
+
+export const useAccessibilityScreenReader = () => {
```

Comment:
I think you can move this to global hooks because maybe in the future it will be useful in other components because I plan to improve accessibility